ui/src/app/nav.tsx

```diff
@@ -18,7 +18,11 @@ function SubmitCSR({ csrText, onClickFunc }: { csrText: string, onClickFunc: any
         extractCSR(csrText.trim())
         csrIsValid = true
     }
-    catch { }
+    catch (e) {
+        if (e instanceof Error) {
+            console.log(e.message)
+        }
+    }
```

Comment:
There is no point in logging this, it will just spam the console with the same generic `couldn't process csr` error messages since this runs every time the text changes. We already return a different component when it fails.

Reply:
✅